Makefile

```diff
@@ -0,0 +1,1398 @@
+# Makefile.in generated by automake 1.14.1 from Makefile.am.
```

Comment:
You should not change and check in the Makefile directly, instead, please add you changes into Makefile.am, then using ./configure to generate the Makefile.